src/main/java/com/example/cabbooking/servicetest/DriverService.java

```diff
@@ -1,8 +1,5 @@
 package com.example.cabbooking.servicetest;
 
```

Comment:
Note that the file names that you have provided should be like` DriverServiceTest  .`

Also the junits should follow https://www.baeldung.com/java-unit-testing-best-practices write more test cases bothe positive and negative cases like in case of exceptions also.

Reply:
yes I will added more test case for both positive and negative.